### ISSUE TITLE
./ava Memory Leak

### ISSUE THREAD

**Issue by rkass** [ava-labs/gecko#214]
## Issue

Running `./ava  > /dev/null` overnight causes my computer to grind to a hault. Looking in activity monitor I see ava consuming 10GB of RAM. Note that I have finished bootstrapping and am currently running a staked validator.

## System Specs

Macbook Air, macOS Mojava 10.14.6, 8GB RAM, intel core i5

**Comment by rkass** [comment]
Seems related to https://github.com/ava-labs/gecko/issues/210

**Comment by aaronbuchwald** [comment]
Hey @rkass thanks for reporting the issue here, could you provide any more details about your setup? You should be able to find all the logs in the directory ~/.gecko/logs, would you mind sharing those?

**Comment by rkass** [comment]
cat ~/.gecko/logs/node/chain/11111111111111111111111111111111LpoYY/0.log
....
a bunch of
```
WARN [06-06|06:08:30] <chain 11111111111111111111111111111111LpoYY> /snow/engine/snowman/transitive.go#188: GetFailed called without sending the corresponding Get message from LQwRLm4cbJ7T2kxcxp4uXCU5XD8DFrE1C
```

cat ~/.gecko/logs/node/0.log

A bunch of 
```
INFO [06-06|04:50:46] /network/peer.go#380: peer attempting to connect with newer version avalanche/0.5.1. You may want to update your client
```

**Comment by rkass** [comment]
Seems mostly resolved in 0.5.3. After 36 hours of running only consuming 600 mb of ram
